margins.go

```diff
@@ -62,3 +74,38 @@ func (c *Client) GetOrderMargins(orderParams []OrderMarginParam) ([]OrderMargins
 
 	return out, nil
 }
+
+func (c *Client) GetBasketMargins(orderParams []OrderMarginParam, considerPositions bool, compactMode bool) (BaksetMargins, error) {
```

Comment:
Can we change the input signature for GetBasketMargins and GetMargins to a struct instead of multiple args?

Ref: 
https://github.com/zerodha/gokiteconnect/blob/develop/portfolio.go#L144

Something like this?

```go
type GetMarginParams struct {
  OrderParams []OrderMarginParam
  Compact bool
}

func (c *Client) GetOrderMargins(gp GetMarginParams) ([]OrderMargins, error) {
...
```

Reply:
Done